<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="calendar_picker"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="date_frame">
      <item>
       <spacer name="date_spacer_L">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="date_box">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="date_label">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Date dd of mmmm yyyy  hh:mm:ss</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="date_spacer_R">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="button_frame">
      <item>
       <widget class="QPushButton" name="add_15m">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+ 15 min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_1h">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+ 1h</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_rest">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Rest</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="verticalFrame">
        <layout class="QVBoxLayout" name="custom_frame">
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Georgia</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Add Custom Time</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSpinBox" name="add_years_vbox">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="specialValueText">
              <string/>
             </property>
             <property name="suffix">
              <string> y</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>-10000</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="add_days_vbox">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>8</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="suffix">
              <string> d</string>
             </property>
             <property name="minimum">
              <number>-303</number>
             </property>
             <property name="maximum">
              <number>303</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="add_hours_vbox">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="suffix">
              <string> h</string>
             </property>
             <property name="minimum">
              <number>-23</number>
             </property>
             <property name="maximum">
              <number>23</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="add_minutes_vbox">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="suffix">
              <string> min</string>
             </property>
             <property name="minimum">
              <number>-59</number>
             </property>
             <property name="maximum">
              <number>59</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="add_seconds_vbox">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="suffix">
              <string> s</string>
             </property>
             <property name="minimum">
              <number>-59</number>
             </property>
             <property name="maximum">
              <number>59</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_custom">
           <property name="text">
            <string>+ custom</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="event_grid">
      <item row="0" column="0">
       <widget class="QLineEdit" name="event_text"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="event_accept">
        <property name="text">
         <string>Add Event</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="event_buttons">
        <item>
         <widget class="QPushButton" name="load_events">
          <property name="text">
           <string>Save Events</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_events">
          <property name="text">
           <string>Load Events</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="event_button_spacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableWidget" name="event_table"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
